tests/unit_tests/test_wave_06.py

```diff
@@ -227,11 +253,82 @@ def test_swap_best_by_category_no_other_match_is_false():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    assert not result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_c in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_f in jesse.inventory
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That result is falsy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories
+
+
+
+
+# unit tests for optional part of project
+
+def test_get_by_age_default_age():
+    electronic = Electronics()
+    assert electronic.age == None
```

Comment:
Nice! This works, though it is more customary to use `is` as opposed to `==` to check if something is `None`.